dags/utils/dataproc.py

```diff
@@ -66,6 +66,12 @@ def create_cluster(self):
                 if value is not None:
                     properties["core:fs.s3a." + key] = value
 
+        init_actions_uris = []
+        if self.artifact_bucket:
+            init_actions_uris.append(
+                'gs://{}/bootstrap/dataproc_init.sh'.format(self.artifact_bucket)
+            )
```

Comment:
r? @haroldwoo

Reply:
I am addressing this in a PR i have for review soon: https://github.com/mozilla/telemetry-airflow/pull/620/files.

By adding init_action_uris as a parameter rather than relying on self.artifact_bucket being set

Reply:
This is fine, when I commit my PR ill make sure that your code changes to use an empty array for init_action_urls